instrumentation/shoryuken/opentelemetry-instrumentation-shoryuken.gemspec

```
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+lib = File.expand_path('lib', __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'opentelemetry/instrumentation/shoryuken/version'
+
+Gem::Specification.new do |spec|
+  spec.name        = 'opentelemetry-instrumentation-shoryuken'
+  spec.version     = OpenTelemetry::Instrumentation::Shoryuken::VERSION
+  spec.authors     = ['OpenTelemetry Authors']
+  spec.email       = ['cncf-opentelemetry-contributors@lists.cncf.io']
+
+  spec.summary     = 'Shoryuken instrumentation for the OpenTelemetry framework'
+  spec.description = 'Shoryuken instrumentation for the OpenTelemetry framework'
+  spec.homepage    = 'https://github.com/open-telemetry/opentelemetry-ruby-contrib'
+  spec.license     = 'Apache-2.0'
+
+  spec.files = Dir.glob('lib/**/*.rb') +
+               Dir.glob('*.md') +
+               ['LICENSE', '.yardopts']
+  spec.require_paths = ['lib']
+  spec.required_ruby_version = '>= 3.0'
+
+  spec.add_dependency 'opentelemetry-api', '~> 1.0'
+  spec.add_dependency 'opentelemetry-common', '~> 0.20.0'
+  spec.add_dependency 'opentelemetry-instrumentation-aws_sdk', '~> 0.5.0'
+  spec.add_dependency 'opentelemetry-instrumentation-base', '~> 0.22.1'
+
+  spec.add_development_dependency 'activejob', '>= 6.0'
+  spec.add_development_dependency 'appraisal', '~> 2.2.0'
+  spec.add_development_dependency 'aws-sdk-sqs', '>= 1.55.0'
+  spec.add_development_dependency 'bundler', '~> 2.4'
+  spec.add_development_dependency 'minitest', '~> 5.0'
+  spec.add_development_dependency 'opentelemetry-sdk', '~> 1.1'
+  spec.add_development_dependency 'opentelemetry-test-helpers', '~> 0.3'
+  spec.add_development_dependency 'rspec-mocks'
+  spec.add_development_dependency 'rubocop', '~> 1.54.1'
```

@mcinbell  - After merging `main` into your branch, the CI failed because we recently added `rubocop-performance` to the repo's main `rubocop.yml`. This suggestion should allow `rubocop` to run on the shoryuken job.

This suggestion also bumps `rubocop` to the latest version.

```suggestion
  spec.add_development_dependency 'rubocop', '~> 1.59.0'
  spec.add_development_dependency 'rubocop-performance', '~> 1.19.1'
```